src/index.html

```diff
@@ -2,11 +2,408 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        class="logo"
+        src="images/logo.png"
+        alt="moyo logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              ФОТО
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              ВИДЕО
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="wrapper">
+      <div class="catalog">
+        <div class="product-card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            class="product-card__image"
+            alt="iMac"
+          >
+
+          <div class="product-card__description description">
+            <p class="description__title">
+              Моноблок APPLE A1419 iMac 27&#34; Retina 5K (MNED2UA/A)
+            </p>
+            <p class="description__code">
+              Код товара: 195434
+            </p>
+            <div class="description__rating rating">
+              <div class="rating__rate-stars rate-stars">
```

Comment:
You should have `rate-stars--4` modifier here.

```suggestion
              <div class="rating__rate-stars rate-stars rate-stars--4">
```